.gitignore

```diff
@@ -6,6 +6,7 @@
 .project
 .classpath
 .settings
+*.iml
```

Comment:
I do not know where this came from, but if you look at [this .gitignore](https://github.com/roboconf/roboconf-platform/blob/master/.gitignore), there is also an **.idea/** directory. Maybe it is worth adding it (I have not yet used IntelliJ).